website/pages/downloads/index.jsx

```diff
@@ -1,25 +1,102 @@
 import VERSION from 'data/version.js'
-import ProductDownloader from '@hashicorp/react-product-downloader'
 import Head from 'next/head'
 import HashiHead from '@hashicorp/react-head'
+import ProductDownloader from '@hashicorp/react-product-downloader'
+import styles from './style.module.css'
+import logo from '@hashicorp/mktg-assets/dist/product/packer-logo/color.svg'
 
-export default function DownloadsPage({ releaseData }) {
+export default function DownloadsPage({ releases }) {
   return (
-    <div id="p-downloads" className="g-container">
-      <HashiHead is={Head} title="Downloads | Packer by HashiCorp" />
+    <>
+      <HashiHead is={Head} title={`Downloads | Packer by HashiCorp`} />
+
       <ProductDownloader
-        product="Packer"
-        version={VERSION}
-        releaseData={releaseData}
+        releases={releases}
+        packageManagers={[
+          {
+            label: 'Homebrew',
+            commands: [
+              'brew tap hashicorp/tap',
+              'brew install hashicorp/tap/packer',
+            ],
+            os: 'darwin',
+          },
+          {
+            label: 'Ubuntu/Debian',
+            commands: [
+              'curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -',
+              'sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"',
+              'sudo apt-get update && sudo apt-get install packer',
+            ],
+            os: 'linux',
+          },
+          {
+            label: 'CentOS/RHEL',
+            commands: [
+              'sudo yum install -y yum-utils',
+              'sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo',
+              'sudo yum -y install packer',
+            ],
+            os: 'linux',
+          },
+          {
+            label: 'Fedora',
+            commands: [
+              'sudo dnf install -y dnf-plugins-core',
+              'sudo dnf config-manager --add-repo https://rpm.releases.hashicorp.com/fedora/hashicorp.repo',
+              'sudo dnf -y install packer',
+            ],
+            os: 'linux',
+          },
+          {
+            label: 'Amazon Linux',
+            commands: [
+              'sudo yum install -y yum-utils',
+              'sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo',
+              'sudo yum -y install packer',
+            ],
+            os: 'linux',
+          },
```

Comment:
Can we move this out to the `version` data file like it is here?

https://github.com/hashicorp/waypoint/blob/main/website/data/version.js#L6

Reply:
> Ah ok - I adjusted the title to reflect this. Sorry, when I was tagged I thought it was ready for review, but I see now that you were not the one who tagged me haha

Good call - I've updated the title on the other PRs as well.